app/api-fetch/song-by-id.ts

```
@@ -0,0 +1,59 @@
+        artists: z.array(z.object({
+            id: z.string(),
+            name: z.string(),
+            avatarurl: z.string(),
+        })
+        )
+    })
+});
+
+export default async function fetchSongById(id: string) {
+    try {
+        if (localStorage.getItem("song-" + id) !== null && localStorage.getItem("songTime-" + id) === null) {
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)